pages/home.jsx

```diff
@@ -96,11 +96,15 @@ var BlogSection = React.createClass({
       featuredPost: {
         title: "",
         author: "",
-        publishedDate: "",
+        publishedDate: new Date(),
         contentSnippet: "",
-        link: ""
+        link: "https://blog.webmaker.org"
       },
-      latestPosts: []
+      latestPosts: [{
+        title: "",
+        publishedDate: new Date(),
+        link: "https://blog.webmaker.org"
+      }]
     }
```

Comment:
cool, this will be nice for non-JS users too maybe, so that at least there will be a link to the blog they can click on?